CMakeLists.txt

```diff
@@ -1,56 +1,45 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 3.1)
-PROJECT(state_msgs)
+PROJECT(whole_body_state_msgs)
 
 # Set up project properties
-SET(PROJECT_NAME state_msgs)
-SET(PROJECT_DESCRIPTION "State ROS messages")
+SET(PROJECT_NAME whole_body_state_msgs)
+SET(PROJECT_DESCRIPTION "Whole-body State ROS messages")
 
-# Print initial message
+# Print initial message - using STATUS so it doesn't trigger as a warning
 MESSAGE(STATUS "${PROJECT_DESCRIPTION}, version ${PROJECT_VERSION}")
-MESSAGE("Copyright (C) 2020 University of Edinburgh")
-MESSAGE("All rights reserved.")
-MESSAGE("Released under the BSD 3-Clause License.")
-
-# Set a default build type to 'Release' if none was specified
-IF(NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
-  MESSAGE(STATUS "Setting build type to 'Release' as none was specified.")
-  SET(CMAKE_BUILD_TYPE Release CACHE STRING "Choose the type of build." FORCE)
-  # Set the possible values of build type for cmake-gui
-  SET_PROPERTY(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS "Debug" "Release" "MinSizeRel" "RelWithDebInfo")
-ENDIF()
+MESSAGE(STATUS "Copyright (C) 2020 University of Edinburgh")
+MESSAGE(STATUS "All rights reserved.")
+MESSAGE(STATUS "Released under the BSD 3-Clause License.")
 
 # Find required packages
 FIND_PACKAGE(catkin REQUIRED COMPONENTS
-  roscpp
   rospy
   std_msgs
+  sensor_msgs
```

Comment:
Why is this needed? are you sure?

Reply:
Thanks for noticing. My bad - I thought of `sensor_msgs/JointState` and did overlook there is a `JointState` in this package. Now removed.

Reply:
No problem :)
Indeed, I imagined some like that